lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/179d_ashrae_90_1_2007/179d_ashrae_90_1_2007.SpaceType.rb

```diff
@@ -1,26 +1,49 @@
 class ACM179dASHRAE9012007
   # @!group SpaceType
 
+  OFFICE_SPACE_TYPES_NAMES_MAP = {
+    'SmallOffice' => 'WholeBuilding - Sm Office',
+    'MediumOffice' => 'WholeBuilding - Md Office',
+    'LargeOffice' => 'WholeBuilding - Lg Office'
+  }
+
+  def whole_building_space_type_name(model, primary_building_type)
+    unless ['Office', 'SmallOffice', 'MediumOffice', 'LargeOffice'].include?(primary_building_type)
+      return 'WholeBuilding'
+    end
+
+    floor_area_m2 = model.getBuilding.floorArea
+    # This turns Office in SmallOffice, MediumOffice, LargeOffice
+    granular_bt = model_remap_office(model, floor_area_m2)
+    return OFFICE_SPACE_TYPES_NAMES_MAP[granular_bt]
+  end
```

Comment:
Since our lookup frpm the spreadsheet is building_type = 'Office'  and space_type = 'WholeBuilding - [Sm, Md, Lg] Office', I have to accomodate this...